include/ginkgo/core/distributed/matrix.hpp

```diff
@@ -339,6 +355,8 @@ class Matrix
         ptr_param<const Partition<local_index_type, global_index_type>>
             col_partition);
 
+    dim<2> get_local_size() const override { return local_mtx_->get_size(); }
+
```

Comment:
If we add this we should also have 
```suggestion
    dim<2> get_local_size() const override { return local_mtx_->get_size(); }
     
    dim<2> get_non_local_size() const override { return non_local_mtx_->get_size(); }

```

Reply:
I was thinking adding it to the same feature class. However, distributed vector does not have this information.
I currently only have get_local_size in vector and matrix. Maybe we can add non local version when we need it afterwards